02/app.js

```diff
@@ -10,14 +10,26 @@ console.log(randomArray);
 
 function createRandomArray() {
     const arr = [];
-    const len = getRandomInteger(1, 10)
+    const len = getRandomInteger(1, 10);
+
     for(let i=0; i<len; i++) {
         arr.push( getRandomInteger(1, 100) );
+
+        console.log('Wyświetlam przy pomocy for' , arr[i]);
     }
 
     return arr;
 }
 
+
 function getRandomInteger(min, max) {
     return Math.round(Math.random() * (max-min) + min);
-}
\ No newline at end of file
+}
+
+
+randomArray.forEach( function(element) {
+    console.log('Wyświetlam przy pomocy forEach' , element)
+});
+
+console.log('ostatni element tablicy to:' , randomArray.slice(-1)[0]);
```

Comment:
Przy tym rozwiązaniu (`.slice()`) tworzysz nową tablicę - jeśli nie potrzebujesz tego to wystarczy `randomArray[randomArray.length -1]`;